[WARNING] - 2022-02-07 07:56:20,765 :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment. in _internal:225
[INFO] - 2022-02-07 07:56:20,766 :  * Running on http://172.18.0.61:5000/ (Press CTRL+C to quit) in _internal:225
[INFO] - 2022-02-07 07:56:20,930 : 172.18.0.1 - - [07/Feb/2022 07:56:20] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:56:24,931 : Successfully received Dutch data from COVID19 API in covid:42
[INFO] - 2022-02-07 07:56:24,984 : 172.18.0.1 - - [07/Feb/2022 07:56:24] "GET /netherlands HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:56:28,641 : Successfully received Dutch data from COVID19 API in covid:42
[INFO] - 2022-02-07 07:56:28,705 : 172.18.0.1 - - [07/Feb/2022 07:56:28] "GET /netherlands HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:56:45,647 : Successfully received Dutch data from COVID19 API in covid:42
[INFO] - 2022-02-07 07:56:45,724 : 172.18.0.1 - - [07/Feb/2022 07:56:45] "GET /netherlands HTTP/1.1" 200 - in _internal:225
[WARNING] - 2022-02-07 07:57:00,704 :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment. in _internal:225
[INFO] - 2022-02-07 07:57:00,704 :  * Running on http://172.18.0.61:5000/ (Press CTRL+C to quit) in _internal:225
[INFO] - 2022-02-07 07:57:00,899 : 172.18.0.1 - - [07/Feb/2022 07:57:00] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:57:39,433 : 172.18.0.1 - - [07/Feb/2022 07:57:39] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:57:39,719 : 172.18.0.1 - - [07/Feb/2022 07:57:39] "GET / HTTP/1.1" 200 - in _internal:225
[WARNING] - 2022-02-07 07:57:42,377 :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment. in _internal:225
[INFO] - 2022-02-07 07:57:42,378 :  * Running on http://172.18.0.61:5000/ (Press CTRL+C to quit) in _internal:225
[INFO] - 2022-02-07 07:57:42,528 : 172.18.0.1 - - [07/Feb/2022 07:57:42] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:57:51,446 : Successfully received Dutch data from COVID19 API in covid:42
[INFO] - 2022-02-07 07:57:51,511 : 172.18.0.1 - - [07/Feb/2022 07:57:51] "GET /netherlands HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:57:54,392 : 172.18.0.1 - - [07/Feb/2022 07:57:54] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:58:01,468 : Successfully received Dutch data from COVID19 API in covid:42
[INFO] - 2022-02-07 07:58:01,548 : 172.18.0.1 - - [07/Feb/2022 07:58:01] "GET /netherlands HTTP/1.1" 200 - in _internal:225
[WARNING] - 2022-02-07 07:59:12,053 :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment. in _internal:225
[INFO] - 2022-02-07 07:59:12,053 :  * Running on http://172.18.0.61:5000/ (Press CTRL+C to quit) in _internal:225
[INFO] - 2022-02-07 07:59:12,214 : 172.18.0.1 - - [07/Feb/2022 07:59:12] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:59:13,869 : 172.18.0.1 - - [07/Feb/2022 07:59:13] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:59:18,927 : Successfully received Dutch data from COVID19 API in covid:39
[INFO] - 2022-02-07 07:59:18,989 : 172.18.0.1 - - [07/Feb/2022 07:59:18] "GET /netherlands HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:59:23,634 : 172.18.0.1 - - [07/Feb/2022 07:59:23] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 07:59:26,955 : Successfully received Dutch data from COVID19 API in covid:39
[INFO] - 2022-02-07 07:59:27,026 : 172.18.0.1 - - [07/Feb/2022 07:59:27] "GET /netherlands HTTP/1.1" 200 - in _internal:225
[WARNING] - 2022-02-07 08:07:49,621 :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment. in _internal:225
[INFO] - 2022-02-07 08:07:49,621 :  * Running on http://172.18.0.61:5000/ (Press CTRL+C to quit) in _internal:225
[INFO] - 2022-02-07 08:07:49,773 : 172.18.0.1 - - [07/Feb/2022 08:07:49] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:07:52,775 : 172.18.0.1 - - [07/Feb/2022 08:07:52] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:07:58,075 : 172.18.0.1 - - [07/Feb/2022 08:07:58] "[33mGET /summart HTTP/1.1[0m" 404 - in _internal:225
[INFO] - 2022-02-07 08:08:00,788 : 172.18.0.1 - - [07/Feb/2022 08:08:00] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:08:04,765 : 172.18.0.1 - - [07/Feb/2022 08:08:04] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:10:11,191 : 172.18.0.1 - - [07/Feb/2022 08:10:11] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:10:48,008 : 172.18.0.1 - - [07/Feb/2022 08:10:48] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:10:50,980 : 172.18.0.1 - - [07/Feb/2022 08:10:50] "GET / HTTP/1.1" 200 - in _internal:225
[WARNING] - 2022-02-07 08:10:53,890 :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment. in _internal:225
[INFO] - 2022-02-07 08:10:53,903 :  * Running on http://172.18.0.61:5000/ (Press CTRL+C to quit) in _internal:225
[INFO] - 2022-02-07 08:10:54,060 : 172.18.0.1 - - [07/Feb/2022 08:10:54] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:10:54,981 : 172.18.0.1 - - [07/Feb/2022 08:10:54] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:10:56,984 : 172.18.0.1 - - [07/Feb/2022 08:10:56] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:10:57,945 : 172.18.0.1 - - [07/Feb/2022 08:10:57] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:11:04,441 : 172.18.0.1 - - [07/Feb/2022 08:11:04] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:12:42,602 : 172.18.0.1 - - [07/Feb/2022 08:12:42] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[ERROR] - 2022-02-07 08:12:42,762 : Exception on /summary [GET] in app:1457
Traceback (most recent call last):
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "main.py", line 29, in serve_summary
    json_template = json.load(open("templates/summary.json"))
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 19 column 5 (char 350)
[INFO] - 2022-02-07 08:12:42,773 : 172.18.0.1 - - [07/Feb/2022 08:12:42] "[35m[1mGET /summary HTTP/1.1[0m" 500 - in _internal:225
[INFO] - 2022-02-07 08:13:00,985 : 172.18.0.1 - - [07/Feb/2022 08:13:00] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[ERROR] - 2022-02-07 08:13:01,180 : Exception on /summary [GET] in app:1457
Traceback (most recent call last):
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "main.py", line 29, in serve_summary
    json_template = json.load(open("templates/summary.json"))
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 20 column 5 (char 373)
[INFO] - 2022-02-07 08:13:01,181 : 172.18.0.1 - - [07/Feb/2022 08:13:01] "[35m[1mGET /summary HTTP/1.1[0m" 500 - in _internal:225
[INFO] - 2022-02-07 08:13:03,057 : 172.18.0.1 - - [07/Feb/2022 08:13:03] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[ERROR] - 2022-02-07 08:13:03,234 : Exception on /summary [GET] in app:1457
Traceback (most recent call last):
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "main.py", line 29, in serve_summary
    json_template = json.load(open("templates/summary.json"))
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 20 column 5 (char 373)
[INFO] - 2022-02-07 08:13:03,236 : 172.18.0.1 - - [07/Feb/2022 08:13:03] "[35m[1mGET /summary HTTP/1.1[0m" 500 - in _internal:225
[WARNING] - 2022-02-07 08:13:10,780 :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment. in _internal:225
[INFO] - 2022-02-07 08:13:10,780 :  * Running on http://172.18.0.61:5000/ (Press CTRL+C to quit) in _internal:225
[INFO] - 2022-02-07 08:13:10,934 : 172.18.0.1 - - [07/Feb/2022 08:13:10] "GET / HTTP/1.1" 200 - in _internal:225
[ERROR] - 2022-02-07 08:13:11,138 : Exception on /summary [GET] in app:1457
Traceback (most recent call last):
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "main.py", line 29, in serve_summary
    json_template = json.load(open("templates/summary.json"))
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 20 column 5 (char 373)
[INFO] - 2022-02-07 08:13:11,141 : 172.18.0.1 - - [07/Feb/2022 08:13:11] "[35m[1mGET /summary HTTP/1.1[0m" 500 - in _internal:225
[ERROR] - 2022-02-07 08:13:15,317 : Exception on /summary [GET] in app:1457
Traceback (most recent call last):
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "main.py", line 29, in serve_summary
    json_template = json.load(open("templates/summary.json"))
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 20 column 5 (char 373)
[INFO] - 2022-02-07 08:13:15,318 : 172.18.0.1 - - [07/Feb/2022 08:13:15] "[35m[1mGET /summary HTTP/1.1[0m" 500 - in _internal:225
[INFO] - 2022-02-07 08:13:17,698 : 172.18.0.1 - - [07/Feb/2022 08:13:17] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[ERROR] - 2022-02-07 08:13:17,911 : Exception on /summary [GET] in app:1457
Traceback (most recent call last):
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "main.py", line 29, in serve_summary
    json_template = json.load(open("templates/summary.json"))
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 20 column 5 (char 373)
[INFO] - 2022-02-07 08:13:17,912 : 172.18.0.1 - - [07/Feb/2022 08:13:17] "[35m[1mGET /summary HTTP/1.1[0m" 500 - in _internal:225
[INFO] - 2022-02-07 08:13:21,288 : 172.18.0.1 - - [07/Feb/2022 08:13:21] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[ERROR] - 2022-02-07 08:13:21,451 : Exception on /summary [GET] in app:1457
Traceback (most recent call last):
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "main.py", line 29, in serve_summary
    json_template = json.load(open("templates/summary.json"))
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting ',' delimiter: line 4 column 3 (char 99)
[INFO] - 2022-02-07 08:13:21,452 : 172.18.0.1 - - [07/Feb/2022 08:13:21] "[35m[1mGET /summary HTTP/1.1[0m" 500 - in _internal:225
[INFO] - 2022-02-07 08:13:26,794 : 172.18.0.1 - - [07/Feb/2022 08:13:26] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[ERROR] - 2022-02-07 08:13:26,976 : Exception on /summary [GET] in app:1457
Traceback (most recent call last):
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "main.py", line 29, in serve_summary
    json_template = json.load(open("templates/summary.json"))
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 20 column 5 (char 373)
[INFO] - 2022-02-07 08:13:26,977 : 172.18.0.1 - - [07/Feb/2022 08:13:26] "[35m[1mGET /summary HTTP/1.1[0m" 500 - in _internal:225
[INFO] - 2022-02-07 08:13:27,555 : 172.18.0.1 - - [07/Feb/2022 08:13:27] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[ERROR] - 2022-02-07 08:13:27,716 : Exception on /summary [GET] in app:1457
Traceback (most recent call last):
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/runner/05-COVID-Dashboard/venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "main.py", line 29, in serve_summary
    json_template = json.load(open("templates/summary.json"))
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/nix/store/wlhmpa2qk786ddp05ahv3jjlds6d44a5-python3-3.8.12/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 20 column 5 (char 373)
[INFO] - 2022-02-07 08:13:27,717 : 172.18.0.1 - - [07/Feb/2022 08:13:27] "[35m[1mGET /summary HTTP/1.1[0m" 500 - in _internal:225
[INFO] - 2022-02-07 08:13:30,346 : 172.18.0.1 - - [07/Feb/2022 08:13:30] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[INFO] - 2022-02-07 08:13:31,236 : 172.18.0.1 - - [07/Feb/2022 08:13:31] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:13:31,256 : 172.18.0.1 - - [07/Feb/2022 08:13:31] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[WARNING] - 2022-02-07 08:13:33,665 :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment. in _internal:225
[INFO] - 2022-02-07 08:13:33,665 :  * Running on http://172.18.0.61:5000/ (Press CTRL+C to quit) in _internal:225
[INFO] - 2022-02-07 08:13:33,815 : 172.18.0.1 - - [07/Feb/2022 08:13:33] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:13:34,594 : 172.18.0.1 - - [07/Feb/2022 08:13:34] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:13:36,186 : 172.18.0.1 - - [07/Feb/2022 08:13:36] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[INFO] - 2022-02-07 08:13:37,161 : 172.18.0.1 - - [07/Feb/2022 08:13:37] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:13:37,536 : 172.18.0.1 - - [07/Feb/2022 08:13:37] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[INFO] - 2022-02-07 08:13:38,407 : 172.18.0.1 - - [07/Feb/2022 08:13:38] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:13:53,336 : 172.18.0.1 - - [07/Feb/2022 08:13:53] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:15:09,956 : 172.18.0.1 - - [07/Feb/2022 08:15:09] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:15:10,803 : 172.18.0.1 - - [07/Feb/2022 08:15:10] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:18:23,705 : 172.18.0.1 - - [07/Feb/2022 08:18:23] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:18:24,551 : 172.18.0.1 - - [07/Feb/2022 08:18:24] "GET /summary HTTP/1.1" 200 - in _internal:225
[WARNING] - 2022-02-07 08:18:34,063 :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment. in _internal:225
[INFO] - 2022-02-07 08:18:34,064 :  * Running on http://172.18.0.61:5000/ (Press CTRL+C to quit) in _internal:225
[INFO] - 2022-02-07 08:18:34,228 : 172.18.0.1 - - [07/Feb/2022 08:18:34] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:18:35,027 : 172.18.0.1 - - [07/Feb/2022 08:18:35] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:18:36,201 : 172.18.0.1 - - [07/Feb/2022 08:18:36] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:18:37,069 : 172.18.0.1 - - [07/Feb/2022 08:18:37] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:23:28,452 : 172.18.0.1 - - [07/Feb/2022 08:23:28] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:23:29,270 : 172.18.0.1 - - [07/Feb/2022 08:23:29] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:23:29,389 : 172.18.0.1 - - [07/Feb/2022 08:23:29] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[INFO] - 2022-02-07 08:23:30,271 : 172.18.0.1 - - [07/Feb/2022 08:23:30] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:23:57,141 : 172.18.0.1 - - [07/Feb/2022 08:23:57] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:23:57,961 : 172.18.0.1 - - [07/Feb/2022 08:23:57] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:24:59,011 : 172.18.0.1 - - [07/Feb/2022 08:24:59] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:24:59,849 : 172.18.0.1 - - [07/Feb/2022 08:24:59] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:25:01,724 : 172.18.0.1 - - [07/Feb/2022 08:25:01] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[INFO] - 2022-02-07 08:25:01,919 : 172.18.0.1 - - [07/Feb/2022 08:25:01] "[36mGET / HTTP/1.1[0m" 304 - in _internal:225
[INFO] - 2022-02-07 08:25:02,605 : 172.18.0.1 - - [07/Feb/2022 08:25:02] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:25:02,752 : 172.18.0.1 - - [07/Feb/2022 08:25:02] "GET /summary HTTP/1.1" 200 - in _internal:225
[WARNING] - 2022-02-07 08:25:05,142 :  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment. in _internal:225
[INFO] - 2022-02-07 08:25:05,142 :  * Running on http://172.18.0.61:5000/ (Press CTRL+C to quit) in _internal:225
[INFO] - 2022-02-07 08:25:05,299 : 172.18.0.1 - - [07/Feb/2022 08:25:05] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:25:06,091 : 172.18.0.1 - - [07/Feb/2022 08:25:06] "GET /summary HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:25:09,064 : 172.18.0.1 - - [07/Feb/2022 08:25:09] "GET / HTTP/1.1" 200 - in _internal:225
[INFO] - 2022-02-07 08:25:09,983 : 172.18.0.1 - - [07/Feb/2022 08:25:09] "GET /summary HTTP/1.1" 200 - in _internal:225
